library/ssl_cli.c

```diff
@@ -2366,9 +2366,16 @@ static int ssl_parse_server_ecdh_params_psa( mbedtls_ssl_context *ssl,
     mbedtls_ssl_handshake_params *handshake = ssl->handshake;
 
     /*
-     * Parse ECC group
+     * struct {
+     *     ECParameters curve_params;
+     *     ECPoint      public;
+     * } ServerECDHParams;
+     *
+     *  1       curve_type (must be named curve)
```

Comment:
`must be **a** named curve`?

Reply:
There [used to be](https://datatracker.ietf.org/doc/html/rfc4492#section-5.4) 3 types of curve:  `explicit_prime`, `explicit_char2`, `named_curve` (and [now there's only one](https://datatracker.ietf.org/doc/html/rfc8422#page17)). So, here I'm trying to talk about the type, not the curve. But I agree the current wording is ambiguous. Would it be better with `curve_type (must be "named_curve")`?

Reply:
Yes, that would be clear, thanks!